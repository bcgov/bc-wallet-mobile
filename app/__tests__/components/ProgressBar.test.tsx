import { render } from '@testing-library/react-native'
import React from 'react'

import ProgressBar from '../../src/components/ProgressBar'

describe('ProgressBar Component', () => {
  test('renders correctly', () => {
    const tree = render(<ProgressBar progressPercent={0} />)
    expect(tree).toMatchSnapshot()
  })
})
