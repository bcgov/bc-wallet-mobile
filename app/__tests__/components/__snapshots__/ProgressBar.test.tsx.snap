// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar Component renders correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#001e3d",
      "height": 11,
      "width": "100%",
    }
  }
>
  <View
    collapsable={false}
    style={
      {
        "backgroundColor": "#FCBA19",
        "bottom": 0,
        "height": "100%",
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
        "transform": [
          {
            "translateX": -375,
          },
          {
            "scaleX": 0,
          },
          {
            "translateX": 375,
          },
        ],
        "width": "100%",
      }
    }
  />
</View>
`;
